src/widgets.js

```diff
@@ -517,6 +517,22 @@ define([
         return widget;
     };
 
+    var media = function (mug, options) {
+        var widget = normal(mug, options);
+        widget.form = "text";
+
+        widget.getValue = function() {
+            return widget.input.val();
+        };
+
+        widget.setValue = function(val) {
+            return widget.input.val(val);
+        };
+
+        widget.mug.form.vellum.initWidget(widget);
```

Comment:
Yeah. this is one of the things I wanted to refactor with media. the name doesn't even tell you what it's doing. `initWidget`? ridiculous (method in core that's only ovewritten by uploader)